src/instances/Form/types.ts

```diff
@@ -0,0 +1,44 @@
+import { Form, FormSchema, StarlightItemResponse } from '../../types'
+
+/**
+ * An Instance that provide methods to request information from a {@link Form}
+ * and its {@apilink Field | Fields}.
+ *
+ * You can access a FormInstance using {@apilink StarlightClient.form} or
+ * using the dynamic syntax on a {@link DynamicStarlightClient}. Usage
```

Comment:
The dynamic syntax here is used on a FormSelector:

```suggestion
 * using the dynamic syntax on a {@link DynamicFormSelector}. Usage
```